format_parser.gemspec

```
@@ -33,8 +33,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'ks', '~> 0.0'
   spec.add_dependency 'exifr', '~> 1', '>= 1.3.8'
   spec.add_dependency 'id3tag', '~> 0.14'
-  spec.add_dependency 'faraday', '~> 0.13'
-  spec.add_dependency 'faraday_middleware', '~> 0.14'
+  spec.add_dependency 'faraday', '~> 1.5.1'
```

```suggestion
  spec.add_dependency 'faraday', '~> 1.5'
```

What about being more flexible about it? Theoretically, minor version bumps don't break compatibility